.github/workflows/tag.yml

```diff
@@ -151,19 +152,22 @@ jobs:
   push_to_chocolatey:
     needs: build_release_assets
     name: Push to Chocolatey
-    runs-on: chocolatey/choco
+    runs-on: windows-latest
     if: github.event_name == 'push' && startsWith(github.event.ref, 'refs/tags')
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Download packages
+      - name: Download nupkg
+        id: download-nupkg
         uses: actions/download-artifact@v4
         with:
-          pattern: os-packages-windows-2022
+          pattern: ggshield.*.nupkg
           path: packages
-          merge-multiple: true
 
       - name: Push to Chocolatey
         run: |
-          scripts/push-to-chocolatey/push-to-chocolatey
+          nupkg_path=$(find ${{ steps.download-nupkg.outputs.download-path }} -maxdepth 1 -type f -name "*.nupkg" | head -n 1)
```

Comment:
I think you must add `shell: bash` to this step, otherwise it's going to run with powershell and your command won't work.

Also, maybe the invocation of the script can be simplified to `scripts/chocolatey/push packages/ggshield.*.nupkg`? (would still need bash, but less moving parts)